src/main/java/com/jcabi/http/mock/MkAnswer.java

```diff
@@ -126,14 +133,14 @@ public Simple(final int status, final String body) {
          * Public ctor.
          * @param status HTTP status
          * @param headers HTTP headers
-         * @param body Body of HTTP response
+         * @param content Body of HTTP response
```

Comment:
@mkordas I changed in order to match the wording elsewhere, content is a binary byte array, and body is that same content represented as a string. As this param is a byte[], I renamed it content. I will update the java doc to say "Body content", to suggest it means both.